transport/http/outbound.go

```diff
@@ -226,11 +226,15 @@ func (o *Outbound) CallOneway(ctx context.Context, treq *transport.Request) (tra
 		return nil, yarpcerrors.InvalidArgumentErrorf("request for http oneway outbound was nil")
 	}
 
-	_, err := o.call(ctx, treq)
+	res, err := o.call(ctx, treq)
```

Comment:
nit: you should add a comment here that we are using res for closing it to avoid a memory leak even if it is empty